Modules/API/API/API.swift

```diff
@@ -6,10 +6,13 @@ import Models
 public class API {
     
     #if DEBUG
-    private let baseURL: String = "https://mobile.track.toggl.space/api/v9/"
+    private static let baseUrl: String = "https://mobile.track.toggl.space"
     #else
-    private let baseURL: String = "https://mobile.track.toggl.com/api/v9/"
+    private static let baseUrl: String = "https://mobile.track.toggl.com"
     #endif
+
+    private let apiUrl: String = "\(baseUrl)/api/v9/"
+    private let reportsUrl: String = "\(baseUrl)/reports/v3/"
```

Comment:
You can use `+` to append strings, which is a little cleaner.
```suggestion
    private let apiUrl: String = baseUrl + "/api/v9/"
    private let reportsUrl: String = baseUrl + "/reports/v3/"
```

Reply:
Concatenation is only needed in language that messed up their string interpolation syntax 🤭